README.md

```diff
@@ -0,0 +1,33 @@
+# gPodder 4 for Sailfish OS
+
+This repository contains the platform-specific customizations and packaging for
+running gPodder 4 on Sailfish OS. Python 3.3.3 and PyOtherSide is already included
+in the device repositories starting with Sailfish OS 1.0.3.8.
+
+## getting started with development
```

Comment:
Headlines should start with uppercase.

Reply:
I'd say *may* here, but I don't really mind changing it.